### DISCUSSION TITLE
Video and Screenshot from the Playwright capabilities available to the plugin Artillery-playwright

### DISCUSSION THREAD

**Discussion by joaormouta** [artilleryio/artillery#1748] [Artillery]
Hi, 
Is it possible to make available these capabilities to the plugin as it is not possible to set them on playwright while doing the load and performance tests?
If it is possible, please help me :)

These capabilities can help the verification of an issue ( bad or not) after a test and it helps also its debug.

Thank you

**Comment by AtefBellaaj** [comment]
Any update on this one?

**Comment by hassy** [comment]
You should be able to take screenshots using the `page.screenshot()` method: https://playwright.dev/docs/screenshots. Note that this will only work when running locally and not on Fargate.

You may also be able to record videos by using this option: https://playwright.dev/docs/api/class-browser#browser-new-context-option-record-video. It will need to be provided to Artillery's Playwright integration via `contextOptions`: https://www.artillery.io/docs/reference/engines/playwright#configuration

**Reply by AtefBellaaj** [reply]
Thank you @hassy  for the reply. I guess the needed use case here is in Fargate, as OP also mentioned "Load Testing". Locally you can also have a headed browser and you can see how the flow is working.
It would be awsome if Artillery can save the screenshots in Aws S3 like it does with the logs.

**Comment by nch3ng** [comment]
+1
